tools/ui-components/src/helpblock/helpblock.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Story } from '@storybook/react';
+import { HelpBlock } from './helpblock';
+import { HelpBlockProps } from './helpblock.types';
+import '../global.css';
+
+const story = {
+  title: 'Example/HelpBlock',
+  component: HelpBlock
+};
+
+const Template: Story<HelpBlockProps> = args => {
+  return <HelpBlock {...args} />;
+};
+
+export const Default = Template.bind({});
+Default.args = {
+  className: 'block mt-5 mb-10 text-[#737373]'
+};
+
+export const LightGray = Template.bind({});
+LightGray.args = {
+  className: 'block mt-5 mb-10 text-[color:var(--gray15)]'
+};
+
```

Comment:
```suggestion
```
There is nothing visible for the help block as story so we should be fine with just keeping the Default Variation.

Reply:
Not sure if we need make it wrap another component. Could look up if there are similar storybook instances where a component is not visible?